### ISSUE TITLE
Create betting pool game

### ISSUE THREAD

**Issue by chrisparsons83** [chrisparsons83/flexspotff#29]

**Comment by chrisparsons83** [comment]
Current status:

NFL Teams and Games for 2022 season are loaded.
Need to build admin interface for setting lines next
Need to build interface for adding lines
Need to build interface for users to make picks
Need to build calculations for processing outcomes
Need to build standings page

**Comment by chrisparsons83** [comment]
Current status now:

- Need to make interface for admin saving lines functional
- Need to build interface for users to make picks
- Need to build calculations for processing outcomes
- Need to build standings page

**Comment by chrisparsons83** [comment]
Currently left:

- Need to build interface for users to make picks
- Need to build calculations for processing outcomes
- Need to build standings page

**Comment by chrisparsons83** [comment]
Currently left:

- Need to show existing bets for people picking
- Need to disable games that have already played
- Need to build calculations for processing outcomes
- Need to build standings page

**Comment by chrisparsons83** [comment]
Currently left:

- Need to build calculations for processing outcomes
- Need to get starting budget dynamically
- Need to build standings page
- Need to show results on old pages

**Comment by chrisparsons83** [comment]
Remaining:

- Need to build standings page
- Need to add step in score calculation for people that have bid in past weeks but not the current week. Likely another table.
- Need to get starting budget dynamically (technically doesn't need to happen until week 2), this involves calcuating past winnings and looking at missing weeks table.
- Need to show results on old pages

**Comment by chrisparsons83** [comment]
Remaining things. None of these need to happen until like week 2, thankfully. Going to switch over to QB challenge setup now and come back to get the rest.

- Need to show results on old pages
- Need to add step in score calculation for people that have bid in past weeks but not the current week. Likely another table.
- Need to get starting budget dynamically (technically doesn't need to happen until week 2), this involves calcuating past winnings and looking at missing weeks table.

**Comment by chrisparsons83** [comment]
K, here are the things that need to be done coming up.

- Deduct 20 points per week for absolutely new people before they submit.
- On a person's first entry past week 1, save their missed weeks in the missed week table.
- When running a scoring week, look for all users that have played in past weeks that did not participate in this week, and add them to the deducted points total.
- Use these numbers towards the total standings.

Need to show results on old pages - made a [separate issue](https://github.com/chrisparsons83/flexspotff/issues/35) for it.

**Comment by chrisparsons83** [comment]
Only thing that's left on this issue is when running a scoring week, look for all users that have played in past weeks that did not participate this week, and add them to the deducted points total.

**Comment by chrisparsons83** [comment]
Closing this out, for now we should be good - any new issues we can add separately.
